mlflow/genai/optimize/optimizers/dspy_mipro_optimizer.py

```diff
@@ -199,14 +200,13 @@ def _display_optimization_result(self, program: "dspy.Predict"):
             initial_score = None
         if initial_score is not None:
             if initial_score == score:
-                _logger.info(
-                    f"Prompt optimization completed. Evaluation score did not change. Score {score}"
-                )
+                _logger.info(f"✅ Optimization complete! Score remained stable at: {score:.2f}")
```

Comment:
Certain environments such as Windows PowerShell don't render emojis. Do we really need to use an emoji here?

Reply:
It's for visual representation and semantically not needed (I don't think there's any case where an emoji is necessary). Indeed, some environments do not support emoji, but we already use emoji in MLflow ([end_run](https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/_tracking_service/client.py#L737-L738)). I wonder what our policy for emojis in logging is. Should we avoid using it? If so, I'm fine with sticking to plain text.